icu4c/source/common/cmemory.h

```diff
@@ -435,7 +512,7 @@ template<typename T, int32_t stackCapacity>
 inline T *MaybeStackArray<T, stackCapacity>::resize(int32_t newCapacity, int32_t length) {
     if(newCapacity>0) {
 #if U_DEBUG && defined(UPRV_MALLOC_COUNT)
-      ::fprintf(::stderr,"MaybeStacArray (resize) alloc %d * %lu\n", newCapacity,sizeof(T));
+        ::fprintf(::stderr, "MaybeStackArray (resize) alloc %d * %lu\n", newCapacity, sizeof(T));
```

Comment:
It's not a good idea to do a trivial bugfix to existing code like this in the same commit as other changes for if problems are encountered with those other changes that lead to them being reverted, the trival bugfix will then be reverted too and that wouldn't benefit anyone.

Reply:
Good point. How would you track trivial bug fixes with ICU tickets? A ticket per little fix, or set of fixes? Do you feel it is reasonable enough to use ticket ICU-21174, "Improvements to ICU internal tools for units work", as a bucket for these?